From 4a45cbab56d4da078fcf5412eac32fe1bbe0c674 Mon Sep 17 00:00:00 2001
From: Brad Bishop <bradleyb@us.ibm.com>
Date: Fri, 22 Apr 2016 19:54:45 -0400
Subject: [PATCH 49/56] Fix a couple incorrect register bases for ast2400

There were a couple copy paste errors in the i2c section of the
ast2400 device tree.

Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 arch/arm/boot/dts/ast2400.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/ast2400.dtsi b/arch/arm/boot/dts/ast2400.dtsi
index 62fab71..2198fd4 100644
--- a/arch/arm/boot/dts/ast2400.dtsi
+++ b/arch/arm/boot/dts/ast2400.dtsi
@@ -215,7 +215,7 @@
 				i2c10: i2c-bus@3c0 {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					reg = <0x380 0x40>;
+					reg = <0x3c0 0x40>;
 					compatible = "aspeed,ast2400-i2c-bus";
 					bus = <10>;
 					clock-frequency = <100000>;
@@ -237,7 +237,7 @@
 				i2c12: i2c-bus@440 {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					reg = <0x400 0x40>;
+					reg = <0x440 0x40>;
 					compatible = "aspeed,ast2400-i2c-bus";
 					bus = <12>;
 					clock-frequency = <100000>;
-- 
1.9.1

